fastlane_core/lib/fastlane_core/command_executor.rb

```diff
@@ -63,7 +63,8 @@ def execute(command: nil, print_all: false, print_command: true, error: nil, pre
               UI.command_output(line)
             end
           end
-        rescue => ex
+        rescue FastlaneCore::FastlanePtyError => ex
```

Comment:
Doesn't this change it, so only this specific exception is caught? What about all others?